src/scripts/requirements.txt

```diff
@@ -0,0 +1 @@
+rdflib~=7.0.0
```

Comment:
really need this?

Reply:
I had deleted it from my local branch, dunno why it still exists in the PR...